plugins/menubar/lib/src/menu_channel.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+import 'menu_item.dart';
+
+// Plugin channel constants. See common/channel_constants.h for details.
+const String _kMenuChannelName = 'flutter/menubar';
+const String _kMenuSetMethod = 'Menubar.SetMenu';
+const String _kMenuItemSelectedCallbackMethod = 'Menubar.SelectedCallback';
+const String _kIdKey = 'id';
+const String _kLabelKey = 'label';
+const String _kEnabledKey = 'enabled';
+const String _kChildrenKey = 'children';
+const String _kDividerKey = 'isDivider';
+
+/// A singleton object that handles the interaction with the menu bar platform
+/// channel.
+class MenuChannel {
+  static const MethodChannel _platformChannel =
```

Comment:
That was sloppy, thanks for catching. I *think* what happened is that I tried to make this const instead of final, got the "only static members can be const" warning, and 'fixed' it by making it static... and then making everything else static for consistency. I choose to believe the fever had already set in when I did that, but I hadn't noticed yet 😬 

Changed to final (the correct fix for that warning), and removed all the statics.